src/sbm_model.jl

```diff
@@ -438,6 +439,22 @@ function update_after_subsurfaceflow(
     return model
 end
 
+function update_total_water_storage(
+    model::Model{N,L,V,R,W,T},
+) where {N,L,V,R,W,T<:SbmModel}
+    @unpack lateral, vertical, network, clock, config = model
+
+    # Update the total water storage based on vertical states
+    # TODO Maybe look at routing in the near future
+    update_total_water_storage(
```

Comment:
I would suggest to resolve TWS here at the `Model` level since all the required info is already here.

Reply:
@verseve, entirely? I kind of like the fact that it follows the same methodology as e.g. 'update_after_subsurfaceflow'.